libc-bottom-half/sources/__wasilibc_populate_args.c

```diff
@@ -0,0 +1,53 @@
+#include <wasi/core.h>
```

Comment:
This filename now seems misleading.    How about `__wasilibc_original_main` or at least something with `main` in it?

Reply:
Good point. How about `__original_main.c`, since that's the main symbol it defines?